src/chrome/content/rules/tomshardware.com.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Certificate mismatch: 
+		- tomshardware.com
+	
+	Expired certificate:
+		- gb.tomshardware.com
+		- dld.tomshardware.com
+-->
+<ruleset name="tomshardware.com">
+	<target host="www.tomshardware.com" />
```

Comment:
Travis is failing since it's missing a target domain in the ruleset that fits the  `<rule from="^http://tomshardware\.com/" to="https://www.tomshardware.com/" />` rule which is `tomshardware.com`, so please add `<target host="tomshardware.com" />` before `<target host="www.tomshardware.com" />`

Reply:
Yeah, you're right. Forgot that. 
Sadly it's still failing. I got no clue what it is.